pkg/cluster/driver/capi-common/capi.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2025, Oracle and/or its affiliates.
 // Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
-package capi
+package capi_common
```

Comment:
can you just call this capi.  Using hyphens for package names is not standard anyway.

Reply:
fixed